gitpod-io/gitpod
# [content-init] remove unneeded chown

This replaces a simple resursive shown to the gitpod user with a correct user mapping. Background is, that when enabling "Feature Preview" we can create contents under `/workspace` that is not owned by `gitpod`. It happens e.g. when running docker containers, the `.docker-root` directory needs to be `root:root`. 
So before this PR you would stop a workspace and after start all files in `/workspace` are owned by `gitpod:gitpod`. After this PR the ownership should be the same as before.

How to test:
 - enable 'Feature Preview'
 - start a workspace and create files/directories under `/workspace` with different users.
 - stop workspace and start it again
 - verify that the ownerships are in tact.

fixes #2899
